test/e2e/metering_manual_install_tests.go

```diff
@@ -107,3 +125,72 @@ func DecodeMeteringConfigManifest(basePath, manifestPath, manifestFilename strin
 
 	return mc, nil
 }
+
+func s3InstallFunc(ctx *deployframework.DeployerCtx) error {
+	// The default ctx.TargetPodsCount value assumes that HDFS
+	// is being used a storage backend, so we need to decrement
+	// that value to ensure we're not going to poll forever waiting
+	// for Pods that will not be created by the metering-ansible-operator
+	ctx.TargetPodsCount = nonHDFSTargetPodCount
+
+	// Before we can create the AWS credentials secret in the ctx.Namespace, we need to ensure
+	// that namespace has been created before attempting to query for a resource that does not exist.
+	_, err := ctx.Client.CoreV1().Namespaces().Get(context.Background(), ctx.Namespace, metav1.GetOptions{})
+	if apierrors.IsNotFound(err) {
+		n := &corev1.Namespace{
+			ObjectMeta: metav1.ObjectMeta{
+				Name: ctx.Namespace,
+				Labels: map[string]string{
+					"name": ctx.Namespace + "-" + testingNamespaceLabel,
+				},
+			},
+		}
+		_, err = ctx.Client.CoreV1().Namespaces().Create(context.Background(), n, metav1.CreateOptions{})
+		if err != nil {
+			return err
+		}
+		ctx.Logger.Debugf("Created the %s namespace", ctx.Namespace)
+	}
+
+	// Attempt to mirror the AWS credentials used to provision the IPI-based AWS cluster
+	// from the kube-system namespace, to the ctx.Namespace that the test context is configured
+	// to use. In the case where this secret containing the base64-encrypted access key id and
+	// secret access key does not exist, replace all instances of the `_` delimiter with `_`,
+	// and create that Secret resource in the ctx.Namespace. In the case where the secret already
+	// exists, exit early.
+	_, err = ctx.Client.CoreV1().Secrets(ctx.Namespace).Get(context.Background(), secretName, metav1.GetOptions{})
+	if apierrors.IsNotFound(err) {
+		s, err := ctx.Client.CoreV1().Secrets("kube-system").Get(context.Background(), secretName, metav1.GetOptions{})
+		if err != nil {
+			return err
+		}
+
+		key, ok := s.Data["aws_access_key_id"]
+		if !ok {
+			return fmt.Errorf("failed to grab the AWS access key ID from the secret data")
+		}
+		access, ok := s.Data["aws_secret_access_key"]
+		if !ok {
+			return fmt.Errorf("failed to grab the AWS secret access key from the secret data")
+		}
+
+		secret := &corev1.Secret{
+			ObjectMeta: metav1.ObjectMeta{
+				Name:      secretName,
+				Namespace: ctx.Namespace,
+			},
+			Data: map[string][]byte{
+				"aws-access-key-id":     key,
```

Comment:
@timflannagan1 the official AWS cred format is `aws_access_key_id`, should we follow suit here?

Reply:
Unfortunately, that's not what we expect to see on the helm-chart level, which is why some translation is needed when decoding the kube-system aws-creds secret (which uses the `aws_access_key_id` data key name): https://github.com/kube-reporting/metering-operator/blob/master/charts/openshift-metering/templates/reporting-operator/reporting-operator-aws-credentials-secrets.yaml#L12

Reply:
That said, I should call out the need to do this translation somewhere in a comment.